nisapi/datasets.yaml

```diff
@@ -1,3 +1,9 @@
 - id: sw5n-wg2p
+  url: https://data.cdc.gov/Flu-Vaccinations/Weekly-Influenza-Vaccination-Coverage-and-Intent-f/sw5n-wg2p/
   vaccine: flu
   start_date: 2021-10-02
+  notes:
+    - One row (national, Pacific Islander, 2023-04-29) has a suppression flag, a null sample size, but a non-null estimate. This row was dropped.
```

Comment:
Again, thanks for documenting these oddities! This certainly underscores a major advantage of having all the work in this repo done and cached.

Reply:
I'm realizing I should also note things like the duplicate rows here